_episodes/03-create.md

```diff
@@ -556,43 +556,54 @@ but it does find the copy in `thesis` that we didn't delete.
 > {: .output}
 >
 > > ## Solution
```

Comment:
I think it should stay after all. Perhaps I was going to add it to the exercise text instead.